docs/madr/decisions/030-extended-global-insights.md

```diff
@@ -0,0 +1,90 @@
+# Extended Global Insights
+
+* Status: accepted
+
+Technical Story: https://github.com/kumahq/kuma/issues/7707
+
+## Context and Problem Statement
+
+We would like to extend current Global Insight to add more meaningful metrics to it. We would like to add info about number of policies, zones,
+services, dataplanes and their status.
+
+## Considered Options
+
+* extend `GlobalInsight` with data from `MeshInsight` and `ZoneInsight`.
+
+## Decision Outcome
+
+We will extend Global Insight with detailed information about number of policies, zones, service, dataplanes and its status. It will be done using 
+information from `MeshInsight` and `ZoneInsight`
+
+### Solution
+
+Right now Global Insight is really simple, it only returns count of resources, example response:
```

Comment:
Currently, `GlobalInsights` is not a real resource. We discussed this with @lahabana, and we think it would be a good idea to make it a real resource. This would allow us to cache the results of the calculations, so that we don't have to recalculate them every time the `/global-insights` endpoint is called. I think it would be good to mention this here.

Reply:
That I didn't know, I thought we are only extending API, thanks for sharing. I've updated the MADR accordingly.

Reply:
I'm not sure it's needed, but I'm not against of doing so.